src/db/schema.py

```diff
@@ -0,0 +1,104 @@
+from typing import Optional, List, Union
+import datetime as dt
+from pydantic import BaseModel
+
+
+class IdOnly(BaseModel):
+    id: int
+
+    class Config:
+        orm_mode = True 
+
+
+class TeamBase(BaseModel):
+    name: str
+
+    class Config:
+        orm_mode = True
+
+
+class TeamCreate(TeamBase):
+    pass
+
+
+class Team(TeamBase):
+    id: int
+
+
+class UserBase(BaseModel):
+    name: str
+    email: str
+    is_active: bool = True
+    team_id: int = None
+
+    class Config:
+        orm_mode = True
+
+
+class UserCreate(UserBase):
+    hashed_password: str
+
+
+class UserBasicInfo(UserBase):
+    id: int
+
+
+class User(UserBasicInfo):
+    main_board_id: int = None
+    favorite_boards: List[IdOnly] = []
```

Comment:
Yes. I tried to figure out how to make it serializer the model as list of ids but couldn't find the solution - do you have an example of how to do so?

Reply:
I think there is no problem with writing it as 
List[int]
its ok

Reply:
It doesn't seems to work since pydantic doesn't know which field from `Board` is the integer
```
In [3]: schema.User.from_orm(user)
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
<ipython-input-3-e180f086179f> in <module>
----> 1 schema.User.from_orm(user)

~/projects/linktious/linktious-server/.venv/lib/python3.8/site-packages/pydantic/main.cpython-38-x86_64-linux-gnu.so in pydantic.main.BaseModel.from_orm()

ValidationError: 1 validation error for User
favorite_boards -> 0
  value is not a valid integer (type=type_error.integer)
```

Reply:
You should add a validator for that: https://pydantic-docs.helpmanual.io/usage/validators/

Reply:
It should look like this?
``
class User(UserBasicInfo):
    main_board_id: int = None
    favorite_boards: List[int] = []

    @validator("favorite_boards", pre=True)
    def favorite_boards_ids(cls, favorite_boards):
        return (board.id for board in favorite_boards)

    class Config:
        orm_mode = True
```

Reply:
yep I think so,
but not sure if i like it.
what do u think?
Also not sure about it performance-wise

Reply:
Probably not very different compare to my previous solution (using IdOnly model)
atleast it  reduce the size of the result